docs/reference/src/main/asciidoc/Weld_-_JSR-299_Reference_Implementation.asciidoc

```diff
@@ -1,6 +1,6 @@
 :numbered!:
 :icons: font
-= Weld - CDI Reference Implementation
+= Weld - CDI Reference Implementation ({weldVersion})
```

Comment:
I find this a bit confusing (a reader may confuse the version as CDI version). What do you think about the following scheme? Weld 2.2.11.Final - CDI Reference Implementation.